app/src/components/Chat.tsx

```diff
@@ -154,12 +158,19 @@ export default function Chat({
       </div>
       <div className="fixed right-4 top-4 flex space-x-4">
         <NavMenu>
-          <LogoutButton supabaseClient={supabaseClient} />
           <NewConversationButton
             createNewConversation={() => {
               newConversation.mutate();
             }}
           />
+          <Button
```

Comment:
Lets make sure to use next/link `Link` for these. You can use `asChild` on `Button` so it renders as the <a> tag and not <button><a>

e.g.

```
<Button
  ...
  asChild
>
  <Link href="/connect">
    <Bluetooth size={20} />
  </Link>
</Button>
```

Reply:
oh and then remove the useRouter if we arent needing it anymore

Reply:
>  I think we could definitely clean up the connect page and make it more of a fleshed out page
totally agree. ESP32 is still fairly unusable - high power consumption, requires soldering a mic on it, want to push out nrf PR and then polish the connect page.